app/src/main/java/com/example/movopfy/features/home/presentation/HomeScreen.kt

```diff
@@ -49,7 +52,11 @@ fun HomeScreen(viewModel: HomeViewModel = koinViewModel(), ) {
             }
 
             is HomeViewModel.HomeUiState.Loaded -> {
-                LazyRowWaitingListToday(list = state.schedules, dimensions = dimensions)
+                LazyRowWaitingListToday(
```

Comment:
Please rename that composable function to SchedulesList

Reply:
+